src/firebase-db.ts

```diff
@@ -227,6 +228,10 @@ export const getAnswer = (embeddableRefId: string): Promise<WrappedDBAnswer | nu
     });
 };
 
+export const getAllAnswersInList = (embeddableRefIds: string[]): Promise<Array<WrappedDBAnswer | null>> => {
+  return Promise.all([...embeddableRefIds.map(getAnswer)]);
```

Comment:
@dougmartin you resolved this one but I don't see any changes. I don't think you need to fix it, but maybe a comment about how it could be optimized if necessary.